client/src/client.rs

```diff
@@ -260,7 +260,7 @@ pub trait RpcApi: Sized {
         include_watchonly: Option<bool>,
     ) -> Result<json::GetTransactionResult> {
         let mut args = [into_json(txid)?, opt_into_json(include_watchonly)?];
-        self.call("getrawtransaction", handle_defaults(&mut args, &[null()]))
+        self.call("gettransaction", handle_defaults(&mut args, &[null()]))
```

Comment:
Doesn't that break compatibility with `<=0.16.3` nodes?

Reply:
I'm pretty sure this was just a typo that no one bumped into... There is `gettransaction` and `getrawtransaction`.